inference/core/utils/container.py

```diff
@@ -0,0 +1,34 @@
+import json
+import socket
+import subprocess
+
+from inference.core.logger import logger
+
+
+def get_container_id():
+    try:
+        return socket.gethostname()
+    except Exception as e:
+        logger.warn(f"Error getting container ID: {e}")
+        return None
+
+
+def get_container_stats():
+    container_id = get_container_id()
+    if container_id:
+        result = subprocess.run(
```

Comment:
for that to work you need to have docker socket exported into the container? Is that ensured somehow?